### DISCUSSION TITLE
目录中去掉目录 以及 章标题上下间距调整

### DISCUSSION THREAD

**Discussion by dubisheng** [tuna/thuthesis#622] [Q&A] [answered]
求教thuthesis中两个问题，

（1）我想把目录里不显示目录这一行。

（2）每一章一开始的章标题上下间距过大我想调整，但是尝试数次均不得法，关于间距这个，我查到cls文件里的是

    \ctexset{%
      chapter = { 
        format     = \centering\sffamily\sanhao,
        beforeskip = 27bp
        afterskip  = 27bp,
      },

但是修改了之后没起到作用。

谢谢大家！

**Comment by zepinglee** [comment, marked as answer]
1. 在重定义 `\tableofcontents` 的位置，使用 `\thu@chapter*[]{\contentsname}` 不会添加到目录。相关的内容参考 `\thu@chapter` 的代码，由于历史原因其中的内容比需求略复杂。

2. 可能是被后面的设置覆盖了。
https://github.com/tuna/thuthesis/blob/1971191972d5cbe826a4d208539465e33f01927d/thuthesis.dtx#L3659-L3664

**Reply by dubisheng** [reply]
谢谢您！我写成这样解决了问题1.

% 目录
%\tableofcontents
\newpage
\chapter*{目\quad 录}
\makeatletter
\@starttoc{toc}
\makeatother
